README.md

```diff
@@ -1,35 +1,142 @@
 ![](https://img.shields.io/badge/Certification%20Level-Community-28A745?link=https://github.com/cyberark/community/blob/master/Conjur/conventions/certification-levels.md)
 
-## CyberArk Ansible Conjur Collection
+# CyberArk Ansible Conjur Collection
 
-****
+This collection contains components to be used with CyberArk Conjur & DAP (Dynamic Access Provider)
+hosted in [Ansible Galaxy](https://galaxy.ansible.com/cyberark/conjur).
 
-### cyberark.conjur
+## Table of Contents
 
-This collection contains plugins to be used for CyberArk Conjur & DAP hosted in [ansible galaxy](https://galaxy.ansible.com/cyberark/conjur).
+* [Requirements](#requirements)
+* [Installation](#installation)
+* [Conjur Ansible Role](#conjur-ansible-role)
+  + [Usage](#usage)
+  + [Role Variables](#role-variables)
+  + [Example Playbook](#example-playbook)
+  + [Summon & Service Managers](#summon---service-managers)
+  + [Recommendations](#recommendations)
+* [Conjur Ansible Lookup Plugin](#conjur-ansible-lookup-plugin)
+  + [Environment variables](#environment-variables)
+  + [Role Variables](#role-variables-1)
+  + [Example Playbook](#example-playbook-1)
+* [Contributing](#contributing)
+* [License](#license)
 
-### Requirements
+<!-- Table of contents generated with markdown-toc
+http://ecotrust-canada.github.io/markdown-toc/ -->
 
-- [CyberArk Conjur Open Source](https://www.conjur.org) v1.x+ or
-- [CyberArk Dynamic Access Provider](https://docs.cyberark.com/Product-Doc/OnlineHelp/AAM-DAP/Latest/en/Content/Resources/_TopNav/cc_Home.htm) v10.x+
-- Ansible v2.9+
+## Requirements
+
+- An instance of [CyberArk Conjur Open Source](https://www.conjur.org) v1.x+ or [CyberArk Dynamic
+  Access
+  Provider](https://docs.cyberark.com/Product-Doc/OnlineHelp/AAM-DAP/Latest/en/Content/Resources/_TopNav/cc_Home.htm)
+  v10.x+ accessible from the target node
+- Ansible >= 2.9
+
+## Installation 
+From terminal, run the following command:
+```sh
+ansible-galaxy collection install cyberark.conjur
+```
+
+## Conjur Ansible Role
+This Ansible role provides the ability to grant Conjur machine identity to a host. Based on that
```

Comment:
can we add a bold **Note** here that this is not yet available in a Galaxy release?